meter/custom/sunspec.yaml

```diff
@@ -0,0 +1,24 @@
+type: sunspec
+params:
+- name: usage
+  choice: [ "grid", "pv", "battery" ]
+  hint: how is the meter installed
+- name: uri
+- name: id
+  default: 1
+# sample: |
```

Comment:
Idee: Der `sample` Block ist egtl. unnötig. Der ergibt sich ja automatisch aus `type` und `params`. Oder haben wir einen Fall wo das nicht ausreichen würde?
Wenn die Annahme stimmt könnte man die Templates aus diesem PR jetzt entfernen und Deinen PR in config sowie den Generator für die README entsprechen umbauen.
`hint` würde ich erstmal grundsätzlich weglassen- Fokus auf Funktion statt UX.

Reply:
Ne passt. Man kännte auch einfach das weiter `sample` nennen anstatt jetzt bei dir `render`;)